source/reference/minio-mc/mc-support-diag.rst

```diff
@@ -28,9 +28,13 @@ The command used to create the diagnostic report has changed over time.
      - Command
      - Notes
 
+   * - Original Command
```

Comment:
Wonder if we should reverse the order of this command name history. With this many changes, it might be easier to understand if the most current were first. Ideas?

Reply:
Some of these are 4+ years old.
I switched the order, but it might be worth considering just dropping it entirely.

It's been an `mc support` command for two years now.